### ISSUE TITLE
[Feature]: Login Termux from proot-distro

### ISSUE THREAD

**Issue by babaric-dev** [termux/proot-distro#249]
### Feature description

We can login proot-distro from Termux app.
Can we do the reverse (login Termux from proot-distro)? I want to use the GUI of proot-distro to develop some scripts for Termux without switching to Termux app everytime.

(Hope my description is clear enough. If not, ask me and I'll edit it.)

**Comment by Grimler91** [comment]
Keep the script in a directory inside proot-distro, or bind mount the directory that has the script with proot distro's `--bind` arg, see [README](https://github.com/termux/proot-distro/blob/master/README.md)

**Comment by babaric-dev** [comment]
True, but environment variables (like `PREFIX`, `HOME`, `TMPDIR`) in proot-distro aren't same as the Termux environment

**Comment by Grimler91** [comment]
I guess it depends on what exactly you want to achieve. 

You can ssh into termux to get a "normal termux shell", which will then use termux's binaries.
